packages/medusa/src/interfaces/payment-service.ts

```diff
@@ -0,0 +1,120 @@
+import { TransactionBaseService } from "./transaction-base-service"
+import {
+  Cart,
+  Customer,
+  Payment,
+  PaymentSession,
+  PaymentSessionStatus,
+} from "../models"
+import { PaymentService } from "medusa-interfaces"
+
+export type Data = Record<string, unknown>
+export type PaymentData = Data
+export type PaymentSessionData = Data
+
+export interface PaymentService<T extends TransactionBaseService<never>>
```

Comment:
thought: I think we might need to have an even sharper differentiation of `payment_session` and the `data` of a `payment_session` i.e. `payment_session.data` since we are mixing the use here.

Reply:
could you give me an example of what you think?